flink-python/src/main/java/org/apache/flink/streaming/api/operators/python/AbstractPythonFunctionOperator.java

```diff
@@ -75,6 +80,7 @@
 	 * A timer that finishes the current bundle after a fixed amount of time.
 	 */
 	private transient ScheduledFuture<?> checkFinishBundleTimer;
+	protected PythonEnvironmentManager environmentManager;
```

Comment:
I agree that it is unnecessary to add it here. 
But the creation of environment manager relies on the information provided by flink operators, such as pythonEnv, temp directory path, dependency manager, and these data is no use for Runner except creating environment manager. Maybe we can initialize the environment manager in the base operator and pass to Runner immediately in open method? 
If move the creation to Runner it will introduce additional concrete class dependency (ProcessEnvironmentManager and DockerEnvironmentManager in the future) of python environment managers.